dd-sdk-android/src/main/java/com/datadog/opentracing/DDSpan.java

```diff
@@ -125,6 +125,18 @@ public DDSpan setError(final boolean error) {
   }
 
   /**
+   * By calling this method the span will be removed from the current active Trace without
+   * actually being persisted.
+   *
+   * Note: This method is meant for internal SDK usage. Be aware that if used this Span will
+   * be removed from the Trace and lost.
+   *
+  */
+  public final void drop() {
```

Comment:
indent seems a bit off

Reply:
It is intentionally like that to align with this Java class which was imported like that. If I lintformat this there will be tons of changes in this class unrelated with this commit. Or were you referring to the `docs` indent ?

Reply:
yes, just indent of the docs is not consistent with indent of the method